.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ on:
 jobs:
   build:
     name: "Build"
-    runs-on: ubuntu-latest
```

Comment:
According to https://github.com/actions/runner-images , `ubuntu-latest` stands for `ubuntu-20.04`. So why change?

Reply:
raised PR for runners https://github.com/actions/runner-images/pull/6766